config/application.rb

```diff
@@ -40,6 +40,9 @@ class Application < Rails::Application
     # define cache store
     config.cache_store = :file_store, Rails.root.join('tmp', "cache_file_store_#{Rails.env}")
```

Comment:
In container context i've used memcached for cache_store.
See: https://github.com/zammad/zammad-docker-compose/blob/master/containers/zammad/docker-entrypoint.sh#L58

Could we use redis here too to have a single solution for all file based caches and sessions stores?

Reply:
From my point of view, yes, that would make sense.

Reply:
Would you mind to extend the pr for that?

Reply:
I am not sure that belongs here because this is not about the cache store.

Also, you could probably just replace the `:file_store` with `:redis_store` in your docker image similar to how it is currently done with memcached. (Which is of course ugly and should be handled differently.)

Also, switching both cache and session store to Redis [might bring a new class of problems](https://stackoverflow.com/a/20216054/3515146), so we should probably also think about how to prevent those.

Reply:
I agree. We should limit the scope of this change to a minimum and instead iterate faster with smaller changes.